omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -38,11 +36,153 @@
                     * object, suitable for use in a URL query string or Ajax request.
                     */
                     location.href = window.location.protocol + "//" + window.location.host +
-                    '/' + OPENMRS_CONTEXT_PATH + '/coreapps/clinicianfacing/patient.page?patientId='
-                    + queryParameters['patientId']; // Causes page to reload
+                        '/' + OPENMRS_CONTEXT_PATH + '/coreapps/clinicianfacing/patient.page?patientId='
+                        + queryParameters['patientId']; // Causes page to reload
                 });
-            </includeIf>
-        </ifMode>
+            </script>
+        </includeIf>
+    </ifMode>
+    <!-- jquery loaded for nonessential purposes. -->
+
+    <style>
```

Comment:
@slubwama The retrospective forms have no inline CSS - so externalise this CSS and work to leverage boostrap styling as much as possible

Reply:
@ssmusoke This styling is specific to this form. if Its put in the general css file may end up affecting the entire  forms

Reply:
done